### ISSUE TITLE
py3 support

### ISSUE THREAD

**Issue by sanjayankur31** [lelutin/scout#30]
Hiya,

Just wondering if scout is py3 compatible? Was just fixing the build in Fedora, thought I'd check and maybe build a py3 version too if I can.

Cheers!

**Comment by lelutin** [comment]
hello! scout was built in python 2.6/2.7 so we'd probably need to change the codebase to make it work with py3.

I'd be interested in having this change done, if you feel up to it. my guess is that since all of the operations are so simple it would probably only require a quick 2to3 run :)

thanks for bringing that up!

**Comment by lelutin** [comment]
I've just tried running scout in python3 inside a venv and I got a big mean traceback. so for now it's not py3 compatible, but I'll see if I can get this in order soon. maybe just migrate to py3 code completely.

btw: thanks for the kick in the issues! I've been asleep for too long on this project and I had even forgotten how to install it :P so far I've added some details in the readme for some missing requirements and some info on how to use pip to install instead of distutils directly. but I'll try to move things some more relatively soon
